akka-remote/src/main/protobuf/ContainerFormats.proto

```diff
@@ -26,4 +26,18 @@ enum PatternType {
 message Selection {
   required PatternType type = 1;
   optional string matcher = 2;
-}
\ No newline at end of file
+}
+
+message Identify {
```

Comment:
@patriknw - I have placed the `Identity` and `ActorIdentity` within the `ContainerFormats.proto` - is this ok?

My reasoning is because the `ContainerFormats.proto` is used for serializing `ActorSelection`.

If this is a wrong place to put, please let me know and I'm happy to move accordingly.